src/validations.ts

```diff
@@ -0,0 +1,156 @@
+import { fullNameIfAvailable } from "./util.ts";
+import { AddressBookEntry, SafeTransaction, TxDetails } from "./models.ts";
+import { COW_TOKEN } from "./constants.ts";
+
+export  function notifyIfNot(condition: boolean, msg: string): boolean {
+  if (!condition) {
+    console.log(msg);
+    return false;
+  }
+  return true;
+}
+export function notifyIfDifferent(
+  real: unknown,
+  expected: unknown,
+  msg: string
+): boolean {
+  if (real !== expected) {
+    console.log(`Expected: ${expected}; is: ${real}.`, msg);
+    return false;
+  }
+  return true;
+}
```

Comment:
This doesn't perform the intended purpose of stopping the final output from appearing if something is bad. If we only care about the error log, we can just remove the output value.
Something we could do for later is having this function also take an `errorStack` vector that collects all error messages. Then in the end we check if the vector is empty to determine that no error was encountered. If there is an error, then we can print out all errors nicely in the interface. (Right now, `console.log` would only output to the web console.)